src/lib/registration/sw_mgmt.rb

```diff
@@ -443,18 +444,22 @@ def self.copy_old_credentials(source_dir)
         ::FileUtils.mkdir_p(dir)
       end
 
-      # check for NCC credentials
-      ncc_file = File.join(source_dir, dir, "NCCcredentials")
-      copy_old_credentials_file(ncc_file)
+      Dir[File.join(source_dir, dir, "*")].each do |path|
```

Comment:
note that this by default skip hidden files starting with "." is it problem here?

Reply:
Hidden credentials file would be very strange, I have never seen that.

Reply:
ok, fine. I just want to point it out.